packages/fe-fpm-writer/src/templates.ts

```diff
@@ -0,0 +1,11 @@
+import { join } from 'path';
+/**
+ * Locates template files relative to the dist folder.
+ * This helps to locate templates when this module is bundled and the dir structure is flattened, maintaining the relative paths.
+ *
+ * @param relativeTemplatePath - optional, the path of the required template relative to the ./templates folder. If not specified the root templates folder is returned.
+ * @returns the path of the template specified or templates root folder
+ */
+export function getTemplatePath(relativeTemplatePath?: string): string {
+    return join(...[__dirname, '../templates'].concat(relativeTemplatePath || []));
```

Comment:
That line looks more complicated then it has to be. Wouldn't the snippet below do the same?
```
export function getTemplatePath(relativeTemplatePath: string = ''): string {
    return join(__dirname, '../templates', relativeTemplatePath);
```